### PR TITLE
Fix tcpdump output file name

### PR DESCRIPTION
**Pull request by pradovic**

### CODE REVIEW

File: scripts/run-md.sh [hunk 1]
```diff
@@ -16,8 +16,9 @@ else
 fi
 
 # run tcpdump in the same directory where other artifacts to be uploaded reside
-TCPDUMP_OUTPUT_FILENAME=$(echo "$MD_FILE" | sha256sum | cut -f1 -d\ ).pcap
-echo "saving packet capture for '$MD_FILE' as $TCP_DUMP_OUTPUT_FILENAME"
+TCPDUMP_OUTPUT_FILENAME=${PWD##*/}.pcap
```
Comment (koivunej) [inline scripts/run-md.sh:19]: I don't know what this does, but `readlink -e` might be better. At minimum this should be in quotes to protect against paths with spaces.

Comment (pradovic) [reply scripts/run-md.sh:19]: I tried to minimize it to just the parent folder, as it appears there will be a `README.md` pre folder. Why do we need sha256 and `MD_FILE` at all in  the tcpdump output file name?

Comment (pradovic) [reply scripts/run-md.sh]: The only thing I see missing might be printing both, I think  this is kind of overkill maybe, as if the folder is run `README.md` is the one that is run, so the filename is always the same.

Comment (koivunej) [reply scripts/run-md.sh]: Output is now looking good. I usually use the readlink together with dirname to get the parent path. The sha256sum idea was for not caring about what the path would had been, all ways would had been mangled similarly.

File: scripts/run-md.sh [hunk 2]
```diff
@@ -16,8 +16,10 @@ else
 fi
 
 # run tcpdump in the same directory where other artifacts to be uploaded reside
-TCPDUMP_OUTPUT_FILENAME=$(echo "$MD_FILE" | sha256sum | cut -f1 -d\ ).pcap
-echo "saving packet capture for '$MD_FILE' as $TCP_DUMP_OUTPUT_FILENAME"
+PARENT_DIR_NAME="${PWD##*/}"
+TCPDUMP_OUTPUT_FILENAME="$PARENT_DIR_NAME".pcap
+#TCPDUMP_OUTPUT_FILENAME=$(echo "$MD_FILE" | sha256sum | cut -f1 -d\ ).pcap
```
Comment (koivunej) [inline scripts/run-md.sh:21]:
```suggestion
```

**Review by koivunej — APPROVED** [review]
Approved with the deletion of the commented line. No point keeping that.

**Pull request merged**